auth_quick_master/models/token.py

```diff
@@ -38,3 +41,20 @@ def get_build_url(self):
         """To be extended"""
         self.ensure_one()
         return None
+
+    def get_public_token(self, build_url, build_id, build_login=None, build_user_id=None):
+        if build_user_id:
+            user = self.env['res.users'].browse(int(build_user_id))
+            build_login = user.login
+        else:
+            user = self.env['res.users'].search([('login', '=', build_login)])
+            build_user_id = user.id
```

Comment:
https://www.youtube.com/watch?v=CBIONjz3yrc

build_user_id is ID in build, not in master database

Reply:
Hmm.. I didn't notice that. Probably I should make changes in auth_quick's controller to pull build's user id, because in our scenario we skip the moment where the user ID is passed from the build

Reply:
We don't really need user_id